### PR TITLE
Fix problems with hds navigation rendering in production build

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
Sub navigation bar rendering issue still remains. Fix for that is under it's way by hds team.

EDIT: Nvm, hds was fast and they already released a fix. It will be included in this pr

### CODE REVIEW

File: src/components/navigation/HaukiNavigation.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
-.navigation-header {
+.navigation-header.navigation-header.navigation-header {
```
Comment (VilleMiekkoja) [inline src/components/navigation/HaukiNavigation.css:1]: Not optimal and pretty, can we find a cleaner fix?

Comment (minevala) [reply src/components/navigation/HaukiNavigation.css:1]: Alternative options:
```css
 .navigation-header {
  --header-background-color: var(--primary-color) !important; // We need to force override with !important
}
```

```css
 header.navigation-header { // We need more specific selector to override the default hds-styles
  --header-background-color: var(--primary-color) !important; // We need to force override with !important
}
```

Comment (minevala) [reply src/components/navigation/HaukiNavigation.css:1]: Would it make sense to make all of the customizations in one place? 
```css
:root {
  --primary-color: #6feed3;
  --header-background-color: var(--primary-color) !important;
}
```

**Review by minevala — COMMENTED** [review]
Nice that the solution is on it's way  👍

File: src/components/navigation/HaukiNavigation.css (hunk 1 above)
Comment (VilleMiekkoja) [reply src/components/navigation/HaukiNavigation.css:1]: > Alternative options:
> 
> ```css
>  .navigation-header {
>   --header-background-color: var(--primary-color) !important; // We need to force override with !important
> }
> ```
> 
> ```css
>  header.navigation-header { // We need more specific selector to override the default hds-styles
>   --header-background-color: var(--primary-color) !important; // We need to force override with !important
> }
> ```

Those would work also, though not sure if they would be cleaner. mm..

Comment (minevala) [reply src/components/navigation/HaukiNavigation.css:1]: I think the most cleanest would be to override them all in on place 🤔

Comment (VilleMiekkoja) [reply src/components/navigation/HaukiNavigation.css:1]: Good point!

Comment (VilleMiekkoja) [reply src/components/navigation/HaukiNavigation.css]: - [ ] Override variables in one place

**Review by minevala — APPROVED** [review]
Nice 👍

**Pull request merged**